src/styles/main.scss

```diff
@@ -1,3 +1,42 @@
-body {
-  margin: 0;
+@import "./normalize.scss";
+@import "./reset.scss";
+
+.snakes {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(3, minmax(300px, 1fr));
+  }
+}
+
+.snakes__snake {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+}
+
+.snake-1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.snake-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.snake-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.snake-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.snake-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.snake-6 {
+  background-color: rgb(0, 0, 0);
 }
```

Comment:
Create one variable for all colors and use cycle for setting background-color to the elements